src/stream-analytics/azext_stream_analytics/_params.py

```diff
@@ -0,0 +1,186 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import validate_file_or_dict
+from .vendored_sdks.streamanalytics.models import (
+    SkuName, OutputStartMode, EventsOutOfOrderPolicy, OutputErrorPolicy, UdfType, CompatibilityLevel
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('stream-analytics job create') as c:
+        c.argument('resource_group', resource_group_name_type)
```

Comment:
This parameter should be named `resource_group_name` in custom.py and then you can omit it in `_params.py` if you like.

Reply:
updated